### DISCUSSION TITLE
Video Donloaded but bluerayed entirely

### DISCUSSION THREAD

**Discussion by ADASHDIVERSIFY** [aclap-dev/video-downloadhelper#1431] [Help and Q&A]
### Your question

Good day everyone,

does Video-DownloadHelper  Extension work anymore ? I just used it to download a video was entirely download but when playing it is entirely messed up with blue-ray, and you can't watch a thing 

### Addon & CoApp version and details

does Video-DownloadHelper  Extension work anymore ? I just used it to download a video was entirely download but when playing it is entirely messed up with blue-ray, and you can't watch a thing

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You should search this forum for discussions of DRM & read some of them.
